rat/ops.py

```diff
@@ -43,7 +43,7 @@ def __str__(self):
 @dataclass
 class Index(Expr):
     names: Tuple[str]
-    shifts: Tuple[Union[str, None]] = None
+    shifts: Tuple[Union[str, None]] = (None,)
```

Comment:
I've changes this line because `all(shift is None for shift in shifts)` didn't work on  `variables.Index`.

Reply:
@bbbales2